examples/src/java/com/twitter/heron/examples/api/AckingTopology.java

```diff
@@ -103,18 +108,20 @@ public void close() {
     }
 
     public void nextTuple() {
-      // We explicitly slow down the spout to avoid the stream mgr to be the bottleneck
+      // Explicitly slow down the spout to prevent the stream manager from being  the bottleneck
       Utils.sleep(1);
 
       final String word = words[rand.nextInt(words.length)];
 
-      // To enable acking, we need to emit tuple with MessageId, which is an object
+      // To enable acking, we need to emit each tuple with a MessageId, which is an Object
```

Comment:
We technically need to create a new message id for every new message. Maybe some explanation about that and why we are doing the way we are doing in this example